<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmEmpleados</class>
 <widget class="QWidget" name="frmEmpleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>850</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>2000</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>234</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>189</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>234</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>73</red>
        <green>77</green>
        <blue>95</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>245</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>232</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>234</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>189</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>234</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>73</red>
        <green>77</green>
        <blue>95</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>245</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>232</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>189</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>73</red>
        <green>77</green>
        <blue>95</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>73</red>
        <green>77</green>
        <blue>95</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>145</red>
        <green>145</green>
        <blue>145</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>245</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>232</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>132</red>
        <green>88</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Información de empleados</string>
  </property>
  <property name="toolTip">
   <string>Eliminar información de empleado</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_filtrar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Filtrar por...</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
background-color: :#494D5F;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../imagenes/imagenes.qrc">
        <normaloff>:/iconos/Filtro.png</normaloff>:/iconos/Filtro.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_buscar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Buscar</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
background-color: :#494D5F;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../imagenes/imagenes.qrc">
        <normaloff>:/iconos/Lupa.png</normaloff>:/iconos/Lupa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6" colspan="4">
    <widget class="QComboBox" name="cbox_departamento">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Departamento*</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="10">
    <widget class="QTableView" name="t">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>500</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QLineEdit" name="txt_apellidos">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Apellidos*</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="9">
    <widget class="QLineEdit" name="txt_buscar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Buscar</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="btn_editar">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Editar información de empleado</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
background-color: #8458B3;
color: rgb(235, 232, 235);

</string>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QDateEdit" name="date_fecha_contratacion">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2023</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl_foto">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../imagenes/imagenes.qrc">:/iconos/JAPI1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_nombre_usuario">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Nombre de usuario&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="txt_nombres">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Nombres*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="btn_regresar">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
background-color: :#494D5F;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../imagenes/imagenes.qrc">
       <normaloff>:/iconos/Flecha.png</normaloff>:/iconos/Flecha.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="txt_salario">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Salario*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="btn_eliminar">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
background-color: #8458B3;
color: rgb(235, 232, 235);
</string>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="4">
    <widget class="QComboBox" name="cbox_trabajo">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Trabajo*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="btn_agregar">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Agregar información de empleado</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
background-color: #8458B3;
color: rgb(235, 232, 235);
</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="txt_telefono">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Telefono*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6" colspan="4">
    <widget class="QLineEdit" name="txt_correo">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Correo Electronico*´</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QComboBox" name="cbox_gerente">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Gerente*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QPushButton" name="btn_limpiar">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Limpiar información seleccionada</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
background-color: #8458B3;
color: rgb(235, 232, 235);
</string>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
